roles/alexandria-v2/defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 # roles/alexandria-v2/defaults/main.yml
 project_name: alexandria-v2
```

Comment:
and project_base default here - please match the default value to the rest of the role defaults